sql/src/test/java/io/crate/testing/SqlExpressions.java

```diff
@@ -74,6 +74,12 @@ public SqlExpressions(Map<QualifiedName, AnalyzedRelation> sources,
         this(sources, fieldResolver, null, User.CRATE_USER);
     }
 
+    public SqlExpressions(Map<QualifiedName, AnalyzedRelation> sources,
```

Comment:
I'd avoid creating too many convenience constructors. These tend to make things *more* tedious to change later instead of the other way around.

Reply:
removed